app/grandchallenge/jqfileupload/widgets/uploader.py

```diff
@@ -309,8 +309,8 @@ def delete(self):
 
 class UploadedAjaxFileList(forms.Field):
     def to_python(self, value):
-        if value is None:
-            value = ""
+        if value is None or value == "None":
```

Comment:
Seems strange, in what situations does this occur?

Reply:
I haven't figured out the exact situation, but I think for empty file uploads (so when multiple is st to False) an empty field would get a `"None"` value rather than `None`.